app/controllers/student/messages_controller.rb

```diff
@@ -0,0 +1,13 @@
+class Student::MessagesController < ApplicationController
+  def index
+    @messages = Message.where(chat_group_params)
+  end
+
+  def create
+  end
+
+  private
```

Comment:
__[RuboCop]__

Layout/EmptyLinesAroundAccessModifier: Keep a blank line before and after `private`.


<!-- Comment by [SideCI](https://sideci.com) -->